content/en/database_monitoring/guide/aurora_autodiscovery.md

```diff
@@ -0,0 +1,240 @@
+Aurora cluster auto discovery is a beta feature. If you have any feedback about this feature, contact Support at support@datadoghq.com.
+{{< /beta-callout >}}
+
+This guide assumes that you have configured [Database Monitoring][1] for your Amazon Aurora DB clusters.
+
+## Before you begin
+
+Supported databases
+: Postgres, MySQL
+
+Supported Agent versions
+: 7.53.0+ (Not yet released)
+
+In order to use this feature now, you need to install the beta version of the Agent. See the installation instructions below.
+
+## Overview
+
+The Datadog Agent supports configuring automatic discovery and monitoring of your aurora clusters. This feature works in lieu of manually configuring the Agent by listing individual database host endpoints.
+
+This is helpful for Aurora clusters with [auto-scaling enabled][6], automatically adjusting the number of read replicas in response to workload variations.
+
+This features leverages the Agent [Autodiscovery feature][4] and lets you define configuration templates for Postgres or MySQL checks, and specify which clusters each check should apply to.
+
+## Enabling aurora cluster discovery
+
+The following outlines the different configuration steps to enable aurora cluster discovery in your Datadog Agent.
+
+### Agent installation
+
+To use this feature, you need to install a beta version of the Agent.
+
+The beta version can be found [here][9]. You can use the Agent installation script to install the correct version by running the following command:
+
+```bash
+DD_API_KEY=${API_KEY} DD_SITE="datadoghq.com" \
+DD_AGENT_DIST_CHANNEL=beta DD_AGENT_MAJOR_VERSION=7 \
+DD_AGENT_MINOR_VERSION=52.0~dbm~aurora~autodiscovery~beta~0.3-1 \
+bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
+```
+
+For more information on installing the Datadog Agent, please read the following [documentation][10].
+
+### AWS permissions
+
+It's required that the Agent has permission to run `rds:DescribeDBClusters` and `rds:DescribeDBInstances` in your AWS account. The recommended way to do this, is to attach an IAM role policy to the EC2 instance where the Agent is running.
+
+An example policy that grants the required permissions is:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "rds:DescribeDBClusters",
+        "rds:DescribeDBInstances"
+      ],
+      "Resource": "arn:aws:rds:<region>:<account>:cluster:<name>"
+    }
+  ]
+}
+```
+
+You can also attach the [AmazonRDSReadOnlyAccess][3] policy.
+
+### Configure aurora
+
+By default, the listener discovers all aurora clusters in the account and region where the Agent is running that have the `datadoghq.com/scrape:true` tag applied. You can also configure the Agent to discover clusters with specific tags.
+
+The tags must be applied to the DB cluster (Role: `Regional clutser`). For more information on tagging RDS resources, see the [AWS documentation][7].
```

Comment:
> (Role: `Regional clutser`)

What does `Role` mean?

Reply:
the role of the instance, it should be on the cluster and not the individual instances,  but maybe that's already clear?

<img width="1072" alt="Screenshot 2024-03-03 at 1 14 33 PM" src="https://github.com/DataDog/documentation/assets/14317240/e9cfebf5-50ae-435a-8156-2d8ec97c03bf">

Reply:
ahh i didn't know this is called `Role` in the console.